test/test_netcdf_output_writer.jl

```diff
@@ -891,4 +932,4 @@ for arch in archs
         test_netcdf_vertically_stretched_grid_output(arch)
         test_netcdf_regular_lat_lon_grid_output(arch)
     end
-end
+end
```

Comment:
we usually have an empty line at the end of file

```suggestion
end

```